becquerel/tools/nndc.py

```diff
@@ -699,6 +700,7 @@ def update(self, **kwargs):
                 raise NNDCInputError(
                     'Decay mode must be one of {}, not {}'.format(
                         WALLET_DECAY_MODE.keys(), kwargs['decay'].lower()))
+            warnings.warn('query kwarg "decay" may not be working on NNDC')
```

Comment:
Do we have a NNDCWarning?

Reply:
No just a `NNDCError`. I personally think custom warnings are not that useful. Haven't really figured out a way how to make use of them. For example it seems to not be possible to catch a certain warning type only.

Reply:
Okay, yeah I agree. Let's include the `kwargs` in this:

```suggestion
            warnings.warn(
                'query kwarg "decay" may not be working on NNDC, kwargs: {}'
                .format(kwargs)
            )
```

Reply:
Do we want this suggestion? @markbandstra @cosama ?

Reply:
It's fine but I don't really see why that extra information is needed. The point is just to flag when a user sets kwarg `"decay"` and notify that the query may work but the behavior probably won't be what they expect, and it's not an issue on our side. The workaround is to select the relevant rows of the returned `DataFrame`. Maybe I'm missing something.

Reply:
Okay, can the warning be expanded to include your comments then? Thanks!